solitude/settings/products.py

```diff
@@ -0,0 +1,20 @@
+# A definiton of Products for Payments for Firefox Accounts. Marketplace
+# defines these dynamically. However for the moment we will define these from
+# configuration files.
+from decimal import Decimal
+
+products = {
```

Comment:
why can't this just be ``BRAINTREE_PRODUCTS`` or something in the normal settings? That way you won't need all that import code.